### PR TITLE
Release 3.4.0

### PR DESCRIPTION
**Pull request by paul-gene**
### Added
- M1 to M2 Stored Card migration tool
  - New `bin/magento` console command to connect to your remote M1 database and potentially copy across customers
    stored cards. This should be run whilst Braintree is in Production mode.
- Kount ENS webhook
  - Allow "suspected fraud" orders in Magento to be accepted or decline by changing status in your Kount portal
- CVV Re-verification for Stored Cards
  - This option can be enabled so that registered Customers need to provide the CVV in order to use a Stored Card
- Information about Apple Pay on-boarding
- Information about Custom Fields

### Fixed
- Level 2/3 Processing data now only used for Credit/Debit card transactions and now includes shipping tax
- Correct state now set on the PayPal Onclick review page
- Bug where PayPal was not using updated shipping address if the customer changed it during checkout
- Bug that stopped Admins creating orders in the backend when Braintree was the only payment method
- API validation check now uses correct Store IDs when a multi-store is being used

### Removed
- Removed old PayPal `payee email` configuration option as it has been deprecated by Braintree

NB - some feature were already approved and PR'd into the master branch.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [master] - 2019-10
+## [3.4.0] - 2019-11-15
 ### Added
+- M1 to M2 Stored Card migration tool
+  - New `bin/magento` console command to connect to your remote M1 database and potentially copy across customers
```
Comment (AidanThreadgold) [inline CHANGELOG.md]: Can we list the command?

File: Console/VaultMigrate.php [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+        foreach ($websites as $website) {
+            foreach ($customers as $customer) {
+                try {
+                    $m2Customer = $this->customerRepository->get($customer['email'], $website->getId());
+
+                    $output->write(
+                        "<info>Customer {$customer['braintree_id']} found in {$website->getName()}...</info>"
+                    );
+
+                    foreach ($customer['storedCards'] as $storedCard) {
```
Comment (AidanThreadgold) [inline Console/VaultMigrate.php]: should this be wrapped in a SQL commit?

File: CHANGELOG.md (hunk 1 above)
Comment (paul-gene) [reply CHANGELOG.md]: New command is outlined in the README :) But yes, I can put it here as well!

File: Console/VaultMigrate.php (hunk 2 above)
Comment (paul-gene) [reply Console/VaultMigrate.php]: As discussed, the repository `save` method utilises transactions, and re-running the command will not result in duplicate data.

**Pull request merged**